### ISSUE TITLE
updated to the latest version, none snippets where found

### ISSUE THREAD

**Issue by jjmaceda** [zaggino/brackets-snippets#26]
Hi, after I updated the plugin this morning, I can't found any snippets, all are gone, tried in a Mac and in a Linux machine same problem, I can't load even the default ones

**Comment by rhbecker** [comment]
Same happened for me. When I pulled up the Brackets extension manager, I noticed it listed this extension as having an error. Then I noticed the extension directory, while still present, was empty. I uninstalled and re-installed, and at superficial first glance, all seems ok.

**Comment by jjmaceda** [comment]
nope that didn't work, tried uninstall/re install, still the snippets list
is empty :/

**Comment by rhbecker** [comment]
Upon further investigation, it seems maybe the code is now requiring the `lang` meta to be set - e.g.
`##lang: js`

When I add that to various snippets and refresh Brackets (via F5), they become available.

I'll say this on an issue related more closely to that meta value: If `lang` is now required, it seems like it should probably recognize / accept an apply-to-all value, such as `*`.

**Comment by jjmaceda** [comment]
Yes, that did the trick,  so the extension need to add the lang meta
automatically for each new extension at least

**Comment by dustinpoissant** [comment]
So is there a fix or no? If not how do a downgrade?

**Comment by zaggino** [comment]
Version 0.6.1 has been published to the registry which fixes this.

**Comment by rhbecker** [comment]
0.6.1 verified to fix this issue for me.
